Sources/SwiftSecurity/Keychain/Keychain.swift

```diff
@@ -233,6 +233,34 @@ extension Keychain: SecItemStore {
         
         return try remove(query)
     }
+
+    public func update<SecItem>(
+        _ value: SecValue<SecItem>,
+        query: SecItemQuery<SecItem>
+    ) throws -> Bool {
+        var query = query
+        query.accessGroup = accessGroup.rawValue
+
+        var updateDict: [String: Any] = [:]
+        switch value {
+        case .data(let data):
+            updateDict[kSecValueData as String] = data
+        case .reference(let reference):
+            updateDict[kSecValueRef as String] = reference
+        default:
```

Comment:
```suggestion
        case .dictionary, .persistentReference:
```
More type-safe + more consistent with `store()`